### PR TITLE
(fetature) Add TimeEntries for other users

### PR DESCRIPTION
**Pull request by peterwillemoeskondrup**
(feature) I was missing the option for creating timeentries for other users in the workspace

### CODE REVIEW

File: Clockify.Net/ClockifyClient.cs [hunk 1]
```diff
@@ -423,6 +423,21 @@ public Task<IRestResponse<TimeEntryDtoImpl>> CreateTimeEntryAsync(string workspa
             return _client.ExecutePostAsync<TimeEntryDtoImpl>(request);
         }
 
+        /// <summary>
+        /// Add a new time entry to workspace foranother user. If end is not sent in request means that stopwatch mode is active, otherwise time entry is manually added.
```
Comment (Morasiu) [inline Clockify.Net/ClockifyClient.cs:427]: Just small typo here  "foranother"

**Comment by Morasiu** [comment]
My test aren't passing. There is a problem with `CreateTimeEntry`. 

`{"message":"Start datetime 2020-12-01T14:57:28Z is greater than end datetime 2020-12-01T14:57:27Z.","code":501}`

Looks like Clockify problem. I'll write to them.

**Comment by Morasiu** [comment]
Okay. I've spoken with Clockify support and I've just pushed test fix :)

Also, I've notices that you are not using your method `CreateTimeEntryForAnotherUserAsync` in tests. Maybe a typo?

**Comment by peterwillemoeskondrup** [comment]
@Morasiu just fixed the typos, sorry about that. Also changed the test a bit to include an endtime.

**Comment by peterwillemoeskondrup** [comment]
Hi @Morasiu 
I don't how it is working (I dont hope the question is too stupid). Do you have plans of merging the changes / branch ?

**Comment by Morasiu** [comment]
Hey.

I will probably merge Pull Request #31 since 31 is upadated version of this one.

**Pull request closed without merging**